Storage/Site.swift

```diff
@@ -74,7 +74,7 @@ public class Site: Identifiable {
      // Sites may have multiple favicons. We'll return the largest.
     public var icon: Favicon?
     public var latestVisit: Visit?
-    public let bookmarked: Bool?
+    public private(set) var bookmarked: Bool?
 
     public convenience init(url: String, title: String) {
         self.init(url: url, title: title, bookmarked: false)
```

Comment:
when you create a Site. It always has `bookmarked` set to false. But unless `bookmared` is set to nil the ASPanel wont check the bookmark status?